charmcraft.yaml

```diff
@@ -0,0 +1,14 @@
+type: charm
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+parts:
+  charm:
+    build-packages:
+      - cargo
+      - rustc
```

Comment:
Why do we need those 2? We shouldn't need them I think since we don't use `cryptography`

Reply:
Removed.